packages/ai/src/chains/operations/operations/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from "./all";
```

Comment:
Why do we have operations/operations? Is "operations" a chain or can be moved upper level? I personally prefer to use flat structure to be an able to see everything with ls -la. Or at least as flat as possible.

Reply:
I don't like here operations/operations/ specifically, don't know how to fix though. Maybe flatten into operations/